config/config.txt

```diff
@@ -289,7 +289,7 @@ CHECK_RANDOMIZER
 
 ## In-game features
 ##Toggle for having jobs load up from the .txt
-# LOAD_JOBS_FROM_TXT
+LOAD_JOBS_FROM_TXT
```

Comment:
You could just remove the job datum if you're not gonna be using it

Reply:
Yeah, this was an old change when I was trying to get config working. Could probably be reverted, it's all config so idk if any of these changes here go over there.